llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -34,6 +34,53 @@ static cl::opt<unsigned> SLPMaxVF(
         "exclusively by SLP vectorizer."),
     cl::Hidden);
 
+InstructionCost
+RISCVTTIImpl::getRISCVInstructionCost(RISCVInstruction Inst, MVT VT,
```

Comment:
What about making it `getRISCVInstructionCost(ArrayRef<RISCVInstruction> Insts, MVT VT, TTI::TargetCostKind CostKind)` and calculate the cost sum of `Insts`?

Reply:
Updated!